src/commissaire_http/dispatcher/__init__.py

```diff
@@ -210,89 +151,38 @@ def dispatch(self, environ, start_response):
             raise DispatcherError(
                 'Bus can not be None when dispatching. '
                 'Please call dispatcher.setup_bus().')
-        route_info = self._router.routematch(environ['PATH_INFO'], environ)
-
-        # If we have valid route_info
-        if route_info:
-            # Split up the route from the route data
-            route, route_data = route_info
-
-            # Get the parameters
-            try:
-                params = self._get_params(environ, route, route_data)
-            except Exception as error:
-                start_response(
-                    '400 Bad Request',
-                    [('content-type', 'text/html')])
-                return [bytes('Bad Request', 'utf8')]
-
-            # method is normally supposed to be the method to be called
-            # but we hijack it for the method that was used over HTTP
-            jsonrpc_msg = {
-                'jsonrpc': '2.0',
-                'id': str(uuid.uuid4()),
-                'method': environ['REQUEST_METHOD'],
-                'params': params,
-            }
 
+        # Add the bus instance to the WSGI environment dictionary.
+        environ['commissaire.bus'] = self._bus
+
+        # Add the routematch results to the WSGI environment dictionary.
+        match_result = self._router.routematch(environ['PATH_INFO'], environ)
+        if match_result is None:
+            start_response(
+                '404 Not Found',
```

Comment:
I agree, but I've not yet found a dictionary of such constants.  Neither in `requests` nor Python's Standard Library, which is quite surprising.